src/main/util.ts

```diff
@@ -0,0 +1,21 @@
+import { URL } from 'url';
+import path from 'path';
+
+let resolveHtmlPath = null;
+
+if (process.env.NODE_ENV === 'development') {
+  const port = process.env.PORT || 1212;
+  resolveHtmlPath = (htmlFileName: string) => {
+    const url = new URL(`http://localhost:${port}`);
+    url.pathname = htmlFileName;
+    return url.href;
+  };
+} else {
+  resolveHtmlPath = (htmlFileName: string) => {
+    return `file://${path.resolve(__dirname, '../renderer/', htmlFileName)}`;
+  };
+}
+
+export default {
```

Comment:
Please use named export here. You can disable prefer default eslint rule if it errors here.